README.adoc

```diff
@@ -388,38 +374,38 @@ that the server runs in.
 
 == Running the application in a Docker container
 
-To containerize the application, you need a `Dockerfile`, which contains a collection of instructions
-defining how a Docker image is built, what files are packaged into it, what commands run when the
-image runs as a container, and so on. You can find a complete `Dockerfile` under the `start` directory.
-This `Dockerfile` packages the `usr` server package into a Docker image that contains a
-pre-configured Open Liberty server.
+To containerize the application, you need a `Dockerfile` file. This file contains a collection of instructions
```

Comment:
We add the extra "file" because of translation. For example, even if "Dockerfile" isn't translated, the word "file" that follows it will be translated. However, because we currently aren't translating the guides, we can remove the second "file" for now and add it back in if we ever want to send the guide to translation later. :)

Reply:
ok, sounds good :)